test/test_models.py

```diff
@@ -591,6 +604,8 @@ def test_classification_model(model_fn, dev):
         "input_shape": (1, 3, 224, 224),
     }
     model_name = model_fn.__name__
+    if dev == torch.device("cuda") and SKIP_BIG_MODEL:
```

Comment:
I believe `dev` [is a string](https://github.com/pytorch/vision/blob/main/test/common_utils.py#L113), hence `dev == "cuda"` might be required. You should confirm with a debugger.

Reply:
Ah yeah, it is indeed a string and I have confirmed it (already update the PR).